```
 Error: failed to build package: unable to run pipeline: exit status 128
2023/02/27 14:19:12 error during command execution: failed to build package: unable to run pipeline: exit status 128
make: *** [Makefile:458: packages/x86_64/dataplaneapi-2.7.2-r0.apk] Error 1
Error: Process completed with exit code 2.
```

The error is not very helpful. though this work on my machine so i dont know what's wrong here

any idea?

--- a/dataplaneapi.yaml
+++ b/dataplaneapi.yaml
@@ -22,7 +22,7 @@ pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/haproxytech/dataplaneapi
-      branch: v${{package.version}}
+      tag: v${{package.version}}
       destination: dataplaneapi
       expected-commit: 3d8f41ff4f5f138a70b8c5627089fac383132c2e
   - runs: |

